torchvision/ops/_utils.py

```diff
@@ -61,3 +61,64 @@ def split_normalization_params(
         else:
             other_params.extend(p for p in module.parameters() if p.requires_grad)
     return norm_params, other_params
+
+
+def set_weight_decay(
+    model: torch.nn.Module,
+    weight_decay: float,
+    norm_weight_decay: Optional[float] = None,
+    norm_classes: Optional[List[type]] = None,
+    bias_weight_decay: Optional[float] = None,
+    custom_keys_weight_decay: Optional[Dict[str, float]] = None,
+):
```

Comment:
This method is a straight extension of `split_normalization_params`. We shouldn't keep both. I think what we could do is "prove" in this PR that the new method works the same as `split_normalization_params` for the right configuration and then remove it. It's currently private so this won't be a problem.

Reply:
yes, but for sake I don't remove this function.

Reply:
Sorry, I don't understand what you mean. Do you intent to remove it later before the PR is merged?

Reply:
yes